### ISSUE TITLE
sh: 1: ./fusibile: Permission denied

### ISSUE THREAD

**Issue by JX-Github-web** [maybeLx/MVSFormerPlusPlus#31]
Run depth map fusion & filter
./fusibile -input_folder ./output_1536x1152/scan4/points_mvsnet/ -p_folder ./output_1536x1152/scan4/points_mvsnet/cams/ -images_folder ./output_1536x1152/scan4/points_mvsnet/images/ --depth_min=0.001 --depth_max=100000 --normal_thresh=360 --disp_thresh=0.1 --num_consistent=2.0 -color_processing
sh: 1: ./fusibile: Permission denied

**Comment by grady0307** [comment]
I met this bug too.😥

**Comment by JX-Github-web** [comment]
Yes.  For me,tried to execute a directory instead of a specific executable file
What should I do to fix this error?

**Comment by fariba87** [comment]
you should use ./fusibile/fusibile instead of /fusibile

**Comment by maybeLx** [comment]
yes, I will change the code.
